service/api/prettyprint.go

```diff
@@ -88,10 +92,10 @@ func (v *Variable) writeTo(buf io.Writer, top, newlines, includeType bool, inden
 			} else if data.Children[0].OnlyAddr {
 				fmt.Fprintf(buf, "0x%x", v.Children[0].Addr)
 			} else {
-				v.Children[0].writeTo(buf, false, newlines, false, indent)
+				v.Children[0].writeTo(buf, false, newlines, !includeType, indent)
```

Comment:
Adding flags to `print` is hard because it accepts any valid expression, also I'm not convinced about its usefulness.